src/PostgREST/Config/Database.hs

```diff
@@ -45,7 +45,8 @@ prefix = "pgrst."
 dbSettingsNames :: [Text]
 dbSettingsNames =
   (prefix <>) <$>
-  ["db_anon_role"
+  ["db_aggregates_enabled"
```

Comment:
This was missing for the [in-db config](https://postgrest.org/en/stable/references/configuration.html#in-database-configuration) to work.

Will refactor it soon so it can be missed.